nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.pipewire.wireplumber;
+in
+{
+  meta.maintainers = [ maintainers.k900 ];
+
+  options = {
+    services.pipewire.wireplumber = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "A modular session / policy manager for PipeWire";
+      };
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.wireplumber;
+        defaultText = literalExpression "pkgs.wireplumber";
+        description = ''
+          The wireplumber derivation to use.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    services.pipewire.media-session.enable = false;
```

Comment:
there (probably?) should be an assertion to prevent mixing the two session managers